python/pyspark/sql/connect/group.py

```diff
@@ -83,6 +83,26 @@ def __init__(
             self._pivot_col = pivot_col
             self._pivot_values = pivot_values
 
+    def __repr__(self) -> str:
+        # since the expressions are not resolved here,
+        # the string represent can be different from vanilla PySpark.
+        grouping_str = ", ".join(str(e._expr) for e in self._grouping_cols)
+        grouping_str = f"grouping expressions: [{grouping_str}]"
+
+        value_str = str(self._df)
+        value_str = value_str[9:]
```

Comment:
and this 9?

Reply:
for this one, we can build it from the schema, let me update